src/main/scala/nodes/learning/BlockLinearMapper.scala

```diff
@@ -127,9 +139,22 @@ class BlockLinearMapper(
   }
 }
 
+/**
+ * Fits a least squares model using block co-ordinate descent with provided
```

Comment:
I actually think this is 'coordinate descent' as per wikipedia :) http://en.wikipedia.org/wiki/Coordinate_descent